src/style.css

```diff
@@ -1,3 +1,88 @@
+/* stylelint-disable-next-line import-notation */
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+
 body {
+  box-sizing: border-box;
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+  justify-content: space-between;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  margin: 0;
```

Comment:
better reset all browser's margins

```
* {
  margin: 0;
  padding: 0;
}
```

Reply:
and add these styles and borde-box to reset.css

Reply:
The requirements for the task stated: don't use * selector (it impacts performance).

Reply:
it's ok to use this selector to reset some browser's styles

Reply:
but use this selector carefully

Reply:
ok)